model/storage-private/src/main/java/org/keycloak/storage/datastore/DefaultExportImportManager.java

```diff
@@ -858,6 +858,51 @@ public void updateRealm(RealmRepresentation rep, RealmModel realm) {
         if (rep.getOfflineSessionMaxLifespanEnabled() != null) realm.setOfflineSessionMaxLifespanEnabled(rep.getOfflineSessionMaxLifespanEnabled());
         if (rep.getOfflineSessionMaxLifespan() != null)
             realm.setOfflineSessionMaxLifespan(rep.getOfflineSessionMaxLifespan());
+        // ===== REALM CLIENT SESSION VALIDATION =====
+         // Values coming from UI (new values)
+        Integer clientIdle = rep.getClientSessionIdleTimeout();
+        Integer clientMax  = rep.getClientSessionMaxLifespan();
+
+        // Resolve realm SSO values (new OR existing)
+        int realmIdle = rep.getSsoSessionIdleTimeout() != null
+                ? rep.getSsoSessionIdleTimeout()
+                : realm.getSsoSessionIdleTimeout();
+
+        int realmMax = rep.getSsoSessionMaxLifespan() != null
+                ? rep.getSsoSessionMaxLifespan()
+                : realm.getSsoSessionMaxLifespan();
+
+        int realmRememberIdle = rep.getSsoSessionIdleTimeoutRememberMe() != null
+                ? rep.getSsoSessionIdleTimeoutRememberMe()
+                : realm.getSsoSessionIdleTimeoutRememberMe();
+
+        int realmRememberMax = rep.getSsoSessionMaxLifespanRememberMe() != null
```

Comment:
`realmRemberIdle` and `realmRememberMax` are only used in the else branch of the if statement, so we can define them there in order to limit there visibility to that scope.